backend/db/utils.py

```diff
@@ -206,9 +226,30 @@ def refresh_derived_tables(
             break
 
 
+# @joeflack4: finally put in connection pooling. (it didn't help the problem I was having)
+# got the code from our friend: https://chat.openai.com/share/0472ba44-0e58-44ad-bbcb-5eabffae3d39
+def set_search_path(dbapi_connection, connection_record):
+    """Set the search_path when connecting."""
+    if SCHEMA:
+        cursor = dbapi_connection.cursor()
+        cursor.execute(f"SET SESSION search_path='{SCHEMA}'")
+        cursor.close()
+
+def get_db_connection(isolation_level='AUTOCOMMIT', schema: str = SCHEMA, local=False) -> Connection:
```

Comment:
## Connection pooling
@Sigfried Was this necessary at this point or a nice-to-have?